tests/test_openjdk.py

```diff
@@ -156,3 +175,71 @@ def test_jdk_extended(
 
     for check in params.expected_err_strings:
         assert check in testout.stderr
+
+
+@pytest.mark.parametrize(
+    "container_per_test",
+    CONTAINER_IMAGES_CASSANDRA,
+    indirect=["container_per_test"],
+)
+def test_jdk_cassandra(container_per_test):
+    """Starts the Cassandra DB and executes some write and read tests
+    using the cassandra-stress
+    """
+
+    cassandra_versions = container_per_test.connection.check_output(
+        "git ls-remote --tags https://gitbox.apache.org/repos/asf/cassandra.git"
+    )
+
+    lines = re.split("\n", cassandra_versions)
+    cassandra_version = ""
+    cassandra_version_order_value = 0
+    for line in lines:
+        x = re.search("cassandra-(\d).(\d).(\d)$", line)
+        if x:
+            order_value = (
+                int(x.group(1)) * 1000 + int(x.group(2))
+            ) * 1000 + int(x.group(3))
+            if order_value > cassandra_version_order_value:
+                cassandra_version = (
+                    x.group(1) + "." + x.group(2) + "." + x.group(3)
+                )
+                cassandra_version_order_value = order_value
```

Comment:
How about using [`pytest_container.Version`](https://github.com/dcermak/pytest_container/blob/e67f174d9e1546a721ccd7e1f7e74c46b1579bf7/src/pytest_container/runtime.py#L64) for this?

You will not have to perform the parsing yourself then and you get the version comparison for "free".

Reply:
done